contracts/Account.sol

```diff
@@ -225,6 +237,31 @@ contract Account is UUPSOwnableUpgradeable, UsingRegistryUpgradeable, Managed, I
         }
     }
 
+    /**
+     * @notice Pauses external access to the contract.
+     * @dev Functions with the `onlyWhenNotPaused` modifier will be paused. This
+     * should be all the non-permissioned (i.e. not `onlyOwner` or * `onlyManager`)
+     * external/public functions.
+     */
+    function pause() external onlyPauser(pauser) {
+        _pause(paused);
+    }
+
+    /**
+     * @notice Unpauses the contract if it was previously paused using `pause()`.
+     */
+    function unpause() external onlyPauser(pauser) {
+        _unpause(paused);
+    }
+
+    /**
+     * @notice Sets that address permissioned to pause/unpause this contract.
+     * @param _pauser The address that can pause/unpause this contract.
+     */
+    function setPauser(address _pauser) external {
```

Comment:
Should this be added to `initialize()`

Reply:
Looks like we didn't do so when e.g. adding new fields to Manager. I'm not opposed to it, but I guess the question is if it's really needed there (since in production `initialize` will not be called anyways).